Makefile

```diff
@@ -0,0 +1,27 @@
+SHELL := /bin/bash
+REV_FILE=.make-rev-check
+
+set-rev:
+	git rev-parse --short HEAD > $(REV_FILE)
+
+images: set-rev
+	./deploy/images/make-image.sh deploy/images/app.Dockerfile "pubgredzone-app:$$(cat $(REV_FILE))"
```

Comment:
The dockerfiles need to be deploy/images/app.Dockerfile, not deploy/images/app/Dockerfile